PennMobile/src/main/res/layout/directory_list_item.xml

```diff
@@ -39,9 +39,7 @@
             android:id="@+id/tv_person_name"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:paddingBottom="2dp"
-            android:textSize="20sp"
-            android:textStyle="bold"
+            android:textSize="17sp"
```

Comment:
If we change it to styles they would still be "hardcoded" because they are different from the rest.